src/internal_events/unix.rs

```diff
@@ -32,7 +32,7 @@ impl<E: std::error::Error> InternalEvent for UnixSocketConnectionFailure<'_, E>
     }
 
     fn emit_metrics(&self) {
-        counter!("unix_socket_connection_failures", 1);
+        counter!("unix_socket_connection_failures_total", 1);
```

Comment:
`connection_errors_total` or `connection_failures_total`? We use "errors" above, but I'm unsure if that's different than what's happening here.